frontend/src/container/TraceDetail/index.tsx

```diff
@@ -48,6 +48,12 @@ function TraceDetail({ response }: TraceDetailProps): JSX.Element {
 		[response],
 	);
 
+	const traceStartTime = useMemo(() => response[0].startTimestampMillis, [
+		response,
+	]);
+
+	const traceEndTime = useMemo(() => response[0].endTimestampMillis, [response]);
```

Comment:
same here ? do we need null checks for the same ?

Reply:
We have the default values (minTime, maxTime) if these values don't exist.